protocol/tests/client_auditlog_test.go

```diff
@@ -0,0 +1 @@
+package tests
```

Comment:
I'm assuming you plan to move tests from `auditlog_test` etc into this module later on?

Reply:
Yes, for instance, please see https://github.com/coniks-sys/coniks-go/pull/192/files#diff-4b27042e76fbd702dc90612e2d9e9372.